IPython/core/tests/test_oinspect.py

```diff
@@ -38,14 +40,14 @@
 # defined, if any code is inserted above, the following line will need to be
 # updated.  Do NOT insert any whitespace between the next line and the function
 # definition below.
-THIS_LINE_NUMBER = 41  # Put here the actual number of this line
+THIS_LINE_NUMBER = 43  # Put here the actual number of this line
```

Comment:
I think this and the bits above are the result of a mistaken merge with another PR that got merged - can you set them back so they're unchanged in this PR?